### PR TITLE
WIP: Doesn't work but getting there

### PR DESCRIPTION
**Pull request by audunmg**
:smile:

### CODE REVIEW

File: PSHostInventory.psm1 [hunk 1]
```diff
@@ -60,11 +60,18 @@ class OpenDCIMInventory : Inventory
...
+    OpenDCIMInventory([string]$Name, [Uri]$APIURI, [String]$APIKey, [String]$APIUser) : base ($Name)
     {
-        $this.APIURI, $this.APIKey = $APIURI, $APIKey
+        $this.APIURI, $this.APIKey, $this.APIUser = $APIURI, $APIKey, $APIUser
+    }
+    Refresh()
+    {
+        $apiuri = $this.APIURI.AbsoluteUri + "/api/v1/device"
+        $devices = (ConvertFrom-Json (Invoke-WebRequest -Headers @{ 'UserID' = $this.APIUser; 'APIKey' = $this.APIKey} ($apiuri) ).Content).device
+q
```
Comment (larsjsol) [inline PSHostInventory.psm1:75]: I'm guessing this (`q`) is a typo?

**Comment by larsjsol** [comment]
Cool! Let me know when you have something working and i'll merge it in :)

File: PSHostInventory.psm1 (hunk 1 above)
Comment (audunmg) [reply PSHostInventory.psm1:75]: Yes, i probably meant :wq

**Comment by audunmg** [comment]
After all these years, I'm sorry to say I won't have time to finish this.

I don't work there anymore, and it's unlikely I'll run into the combination of openDCIM and powershell again, but who knows.

I'll leave it as it is in case it's useful, but I'm closing this pull request since I won't finish it in any reasonable time.

**Comment by larsjsol** [comment]
No worries @audunmg, I appreciate the thought

**Pull request closed without merging**